.gitlab-ci.yml

```diff
@@ -998,3 +998,6 @@ plugin:ci-tactician:
 
 plugin:ci-waterproof:
   extends: .ci-template-flambda
+
+plugin:ci-trakt:
+  extends: .ci-template
```

Comment:
needs to use ci-template-flamba (because elpi is) and declare the dependency on elpi. See "Job dependencies are declared in 2 places' in https://github.com/coq/coq/blob/master/dev/ci/README-users.md